src/test/unit/algorithms/CondProbTableTest.cpp

```diff
@@ -180,13 +183,13 @@ namespace nupic {
 
 
   //----------------------------------------------------------------------
-  void CondProbTableTest::testTable(const string& testName, CondProbTable& table, 
-                                    const vector<vector<Real> > & rows)
+  void testTable(const string& testName, CondProbTable& table, 
+                 const vector<vector<Real> > & rows)
   {
   
     // Test the numRows(), numCols() calls
-    TESTEQUAL(numRows(), table.numRows());
-    TESTEQUAL(numCols(), table.numColumns());
+    EXPECT_EQ(numRows(), table.numRows());
```

Comment:
Well whoever made that alias changed the behavior since `TESTEQUAL` originally would raise an exception immediately if I recall correctly. I think better to use `ASSERT_EQ` here even if that is a functional change.